tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Persistence/Repositories/TagRepositoryTest.cs

```diff
@@ -654,6 +789,8 @@ public void Can_Get_Tags_For_Entity_Type()
             ContentTypeRepository.Save(contentType);
 
             var content1 = ContentBuilder.CreateSimpleContent(contentType);
+            content1.PublishCulture(CultureImpact.Invariant);
+            content1.PublishedState = PublishedState.Publishing;
```

Comment:
I am not sure why this needs to be added here?

Reply:
if `content1.PublishedState = PublishedState.Publishing;` is not present the test content is acculty marked as an unpulished node, therefor the test will fail do to the `Assert.AreEqual(3, result1.Length);` result1.Length will be 0 becouse we dont get tag from unpublished content.